config/default.yml

```diff
@@ -1415,6 +1415,18 @@ Lint/Debugger:
   Enabled: true
   VersionAdded: '0.14'
   VersionChanged: '0.49'
+  AllowedMethods:
```

Comment:
this seems the opposite of Allowed, should this be `DisallowedMethods`? `ReportedMethods`? `DebuggerMethods`? just `Methods`?

Reply:
Yep, you're toally right. I like the name `DebuggerMethods` best, although I'm also fine with "ForbiddenMethods".

Reply:
I also felt like this when implementing... shall we go with `DebuggerMethods` then?

Reply:
Yep.